### PR TITLE
PAYARA-1499 add support element to tree view

### PR DESCRIPTION
**Pull request by Cousjava**

### CODE REVIEW

**Comment by Cousjava** [comment]
jenkins test please

File: appserver/admingui/core/src/main/resources/org/glassfish/admingui/core/Strings.properties [hunk 1]
```diff
@@ -150,6 +150,8 @@ tree.accessLog=Access Log
 tree.clusters=Clusters
 tree.standaloneInstances=Stand-Alone Instances
 
+tree.support=Support
```
Comment (Pandrex247) [inline appserver/admingui/core/src/main/resources/org/glassfish/admingui/core/Strings.properties]: Is there a reason you couldn't put this in the currently empty Strings.properties file in the support module?

File: appserver/admingui/payara-support/src/main/resources/support/pluginTreeNodeSupport.jsf [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+ * and therefore, elected the GPL Version 2 license, then the option applies
+ * only if the new code is made subject to such option by the copyright
+ * holder.
+-->
+
+<!-- pluginTreeNodeAdminService.jsf -->
+
+<sun:treeNode id="support"
+    text="$resource{i18n.tree.support}"
+    url="/support/support.jsf?configName=${configName}"
```
Comment (Pandrex247) [inline appserver/admingui/payara-support/src/main/resources/support/pluginTreeNodeSupport.jsf]: This URL isn't correct, it needs to be: /payara_support/support/support.jsf
The URL takes the form of: _console-config.xml ID (payara_support) / path / through / folders / to / desired / page.sf_

You also aren't using the config name, so there's no need to pass it through (_?configName=${configName}_).

File: appserver/admingui/payara-support/src/main/resources/support/pluginTreeNodeSupport.jsf [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    imageURL="/resource/images/support.png"
```
Comment (Pandrex247) [inline appserver/admingui/payara-support/src/main/resources/support/pluginTreeNodeSupport.jsf]: This isn't actually pointing to your own image.
Your own image would be at /resource/payara_support/images/support.png

I like the one you've found better though, so leave it as is and remove the one you added.

File: appserver/admingui/payara-support/src/main/resources/support/pluginTreeNodeSupport.jsf (hunk 2 above)
Comment (Cousjava) [inline appserver/admingui/payara-support/src/main/resources/support/pluginTreeNodeSupport.jsf]: The URL will point to wherever Daniel creates the integration, its just a placeholder here.

**Comment by Cousjava** [comment]
jenkins test please

**Comment by Cousjava** [comment]
jenkins test please

**Comment by smillidge** [comment]
@Pandrex247 can you review again?

**Pull request merged**